### PR TITLE
Update Copy, Layout and Header - Bundles Landing Page

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

So there are a few things going on in this PR, because we decided to pull in some of the UX copy/layout changes from a couple of cards. The changes are to the bundles landing page and the 'Guardian Header', which appears on this page and on the monthly contribution checkout and thank you pages.

[Trello card](https://trello.com/c/334eYOYP/512-opportunity-tree-change-header) and [the other one](https://trello.com/c/O9Jqfyrs/517-opportunity-tree-experiment-1-designs-and-spacing).

cc: @superfrank

## Changes

- Trimmed down the introduction, and tweaked the copy and padding.
- Removed the explainer copy from the bundles.
- Tweaked the feature copy and padding on the bundles.
- Removed the divider image that appears between the bundles on sub-desktop breakpoints, and replaced with white space.
- Changed the bundles background colour on mobile to match the introduction background.
- Removed the roundel and nav from the 'Guardian Header'.

## Screenshots

**Desktop:**

desktop-bundles

**Tablet:**

tablet-bundles

**Mobile:**

mobile-bundles

**Checkout Page (Monthly Contributions):**

checkout

### CODE REVIEW

File: frontend/app/views/bundle/bundlesLanding.scala.html [hunk 1]
```diff
@@ -109,19 +95,14 @@ <h2 class="bundles__subheading"><span class="bundles__wrap-line">Become a </span
 									<p>Choose the package you want: Everyday+, Sixday+, Weekend+ and Sunday+</p>
 								</li>
 								<li class="js-digital-benefits">
-									<h3>Digital</h3>
-									<p>All the benefits of the digital subscription</p>
+									<h3>All the benefits of a Digital subscription</h3>
```
Comment (rupertbates) [inline frontend/app/views/bundle/bundlesLanding.scala.html]: Why does Digital have a capital letter here when it doesn't in the section above?

File: frontend/assets/stylesheets/components/_bundle-landing.scss [hunk 2]
```diff
@@ -98,7 +103,7 @@ $why-support-colour: #7cb523;
 	@include mq($from: desktop) {
 		display: flex;
 		flex-direction: column;
-		height: gs-height(10) + $gs-baseline;
+		height: gs-height(9) + ($gs-baseline * 2);
```
Comment (rupertbates) [inline frontend/assets/stylesheets/components/_bundle-landing.scss]: Much more readable ;-)

File: frontend/app/views/bundle/bundlesLanding.scala.html (hunk 1 above)
Comment (Ap0c) [inline frontend/app/views/bundle/bundlesLanding.scala.html]: Good question, because reasons? Just had a word with Nick, he's fine with it for now 🦌.

**Comment by Ap0c** [comment]
Changes are appearing as expected.

**Pull request merged**